Fix main to pass CI

djexvr opened the pull request:
Add openfun config for gitlint to comply with openfun practices.

## Purpose

Gitlint is expecting additional config to pass the CI, this PR should allow the CI to work as expected

djexvr committed:
🔨(gitlint) add openfun gitlint config

Add openfun config for gitlint to comply with openfun practices.

AladdinVR committed:
🎨(lambda) fix linter error

Fix linter issues to pass CI

djexvr committed:
👷(lint-back) fix path to pylintrc

The pylint path is wrong and triggers an error in the lint-back job of
the ci

djexvr committed:
👷(lint-back) fix path to banditrc

The banditrc path is wrong and triggers an error in the lint-back job of
the ci

djexvr committed:
🙈(env) add exception for development.dist

The CI needs the development.dist file to load default env values.
However the env.d directory must remain in the gitignore as it contains
the local env files.

djexvr committed:
🧱(dockerfile) copy gunicorn config

Add the gunicorn config file as it is needed later in the file.

djexvr committed:
🧱(dockerfile) fix docker image entrypoint

The entrypoint of the docker image is wrong causing failure when running
CI

djexvr committed:
📝(changelog) update changelog

Add minor versions related to the fixes provided by this version

djexvr committed:
🩹(lambda) fix flake8 error

Fix a lint error due to a bad configuration of flake8. Since lambda is
deprecated and will be removed in a later version there is no need to
configure it and this simple fix should be enough.

djexvr committed:
👷(lambda-lint) remove lambda lint task

The lambda function is deprecated and will be removed there is no need
to fix failing lint step due to missing config file.

sampaccoud commented on CHANGELOG.md:
```diff
@@ -7,3 +7,13 @@ and this project adheres to [Semantic
+
+## [0.0.1] - 2022-03-18
+
+### Added
+
+- Add openfun's rules to commit format
+
+### Changed
+
+- Fix several bugs in CI to make a clean starting repo
```
this should not be part of the changelog which decribes changes in the application, not in the project tools.
The check-changelog task is not required and we can merge even if it fails. It's just here as a reminder.

djexvr committed:
📝(changelog) revert changelog update

Remove update to chagelog as it concerns update of development tools
rather than functionalities
